task_13/test/test.js

```diff
@@ -0,0 +1,45 @@
+const assert = require('chai').assert
+const {chef, Calculator} = require('../app')
+
+describe('Chef test', ()=>{
+    let cheff = chef
+    
+    it('check if the dish has a valid name', ()=>{
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887245)
```suggestion
    it('check if the dish has a valid name', () =>{
```